src/components/post/PostPreviewModal.jsx

```diff
@@ -12,17 +12,24 @@ function PostPreviewModal(props) {
     imageObjectUrl,
   } = props;
 
+  // Post requires a Post object for rendering
+  const postObj = {
+    id: -1, // arbitrary ID for the Post object
+    imageSrc: imageObjectUrl,
+    content: postContent,
+    postTime: new Date(),
+    username: "username",
+  };
+
   return (
+
     <Modal size="lg" onHide={onHide} show={show} className="post-preview-modal">
-      <Modal.Header closeButton>
+      <Modal.Header>
```

Comment:
Would it be better to keep the closeButton?

Reply:
done